src/awsexplorer/activation.ts

```diff
@@ -92,25 +94,20 @@ async function registerAwsExplorerCommands(
                 outputChannel: lambdaOutputChannel,
                 onRefresh: () => awsExplorer.refresh(node.parent)
             }),
-        telemetryName: {
-            namespace: TelemetryNamespace.Lambda,
-            name: 'delete'
-        }
+        telemetryName: 'Command_aws.deleteLambda'
     })
 
     registerCommand({
         command: 'aws.deleteCloudFormation',
         callback: async (node: CloudFormationStackNode) =>
             await deleteCloudFormation(() => awsExplorer.refresh(node.parent), node),
-        telemetryName: {
-            namespace: TelemetryNamespace.Cloudformation,
-            name: 'delete'
-        }
+        telemetryName: 'Command_aws.deleteCloudFormation'
```

Comment:
Should this not be `cloudformation_delete`?

Reply:
yep, fixed